### PR TITLE
Add section on generated partition column to blog post on hive partitions

### PR DESCRIPTION
**Pull request by keen85**
addressing #387

### CODE REVIEW

**Comment by keen85** [comment]
@MrPowers would you review the PR?

File: src/blog/pros-cons-hive-style-partionining/index.mdx [hunk 1]
```diff
@@ -2,8 +2,8 @@
 title: Pros and cons of Hive-style partitioning
 description: This post discusses the pros and cons of Hive-style partioning.
 thumbnail: ./thumbnail.png
-author: Matthew Powers
-date: 2023-09-27
+author: Matthew Powers, [Martin Bode](https://www.linkedin.com/in/martin-bode) `TODO`
+date: 2023-09-27, updated 2024-04-`TODO`
```
Comment (MrPowers) [inline src/blog/pros-cons-hive-style-partionining/index.mdx:6]: We can just change this to the updated date

File: src/blog/pros-cons-hive-style-partionining/index.mdx [hunk 2]
```diff
@@ -2,8 +2,8 @@
 title: Pros and cons of Hive-style partitioning
 description: This post discusses the pros and cons of Hive-style partioning.
 thumbnail: ./thumbnail.png
-author: Matthew Powers
-date: 2023-09-27
+author: Matthew Powers, [Martin Bode](https://www.linkedin.com/in/martin-bode) `TODO`
```
Comment (MrPowers) [inline src/blog/pros-cons-hive-style-partionining/index.mdx:5]: Can you please add your name here? https://github.com/delta-io/website/blob/5353a54ce5b64017da8cd01d9e8237dd936ff675/src/components/BlogAuthorsName.jsx#L12

File: src/blog/pros-cons-hive-style-partionining/index.mdx [hunk 3]
```diff
@@ -159,6 +159,185 @@ You can bypass these concurrency errors on a Hive-style partitioned table by run
 
 Hive-style partitioning allows users to bypass concurrency issues in certain circumstances, but as we’ve previously mentioned, separating data into subdirectories isn’t strictly necessary to get a full division of data. Logical partitioning (separating data into separate files and recording this in the transaction log) is sufficient and physical partitioning (separate data into separate folders) isn’t strictly necessary.
 
+## Using a generated column to partition time series data
```
Comment (MrPowers) [inline src/blog/pros-cons-hive-style-partionining/index.mdx:162]: Did you intentionally just leave this list of links here?

**Review by MrPowers — COMMENTED** [review]
Left some minor comments.  Will also be interested in having an engineer review this one.

File: src/components/BlogAuthorsName.jsx [hunk 4]
```diff
@@ -38,6 +38,9 @@ export const AUTHORS = {
     profile: "https://www.linkedin.com/in/mshtelma/",
     avatar: MICHAEL_SHTELMA_IMG,
   },
+  "Martin Bode": {
+    profile: "https://www.linkedin.com/in/martin-bode/",
```
Comment (MrPowers) [inline src/components/BlogAuthorsName.jsx:42]: Looks like you're missing the avatar field

**Comment by MrPowers** [comment]
@keen85 - great, this one is ready to merge once we get your avatar in.  Thanks again for this contribution!!

**Review by MrPowers — APPROVED** [review]

File: src/blog/pros-cons-hive-style-partionining/index.mdx [hunk 5]
```diff
@@ -2,8 +2,8 @@
 title: Pros and cons of Hive-style partitioning
 description: This post discusses the pros and cons of Hive-style partioning.
 thumbnail: ./thumbnail.png
-author: Matthew Powers
-date: 2023-09-27
+author: Matthew Powers, Martin Bode
+date: 2023-09-27, updated 2024-04-19
```
Comment (MrPowers) [inline src/blog/pros-cons-hive-style-partionining/index.mdx:6]: @keen85 - can you change this to the updated date?  This is causing the build to fail.

**Comment by MrPowers** [comment]
@keen85 - merged this, thank you!

**Comment by keen85** [comment]
Thanks @MrPowers for carrying me through this 😆

**Pull request merged**